components/FoodCard.js

```diff
@@ -6,15 +6,26 @@ export default function FoodCard({ recipe }) {
   return (
     <Card>
       <RecipeName>{recipe.name}</RecipeName>
-      <Image src={recipe.image} height={300} width={200} alt={recipe.name} />
+      <StyledImage src={recipe.image} height={300} width={200} alt={recipe.name} />
     </Card>
   );
 }
 
 const Card = styled.li`
   list-style: none;
+  margin: 10px;
+  padding: 10px;
 `;
 
 const RecipeName = styled.p`
   font-style: italic;
+  position: relative;
+  text-align: center;
+`;
```

Comment:
This could be a headline instead of a `p`. And for what do you need `position: relative` here? 🤔 

```suggestion
const RecipeName = styled.h3`
  font-style: italic;
  position: relative;
  text-align: center;
`;
```